02-express-tutorial/app.js

```diff
@@ -1,35 +1,54 @@
 const express = require("express");
-const { products } = require("./data");
 const app = express();
+const { products } = require("./data");
+
+
+app.get('/', (req, res) => {
+    res.send('<h1>Home page</h1><a href="/api/products">products</a>')
+})
+
+app.get(" ", (req, res) => {
+    res.send("About page")
+})
 
 app.use(express.static("./public/"));
```

Comment:
do you remember who express will run the code?? cause this middleware might not run on the `get('/')`